dev/fattest.simplicity/src/componenttest/containers/ExternalDockerClientFilter.java

```diff
@@ -171,10 +175,17 @@ public boolean isValid() {
         return valid;
     }
 
+    /*
+     * @return - ip address
+     */
```

Comment:
This actually returns the `host` aka the the full address of the docker host including protocol and port. 
```suggestion
    /*
     * @return - location of remote docker host. Example: tcp://127.0.0.1:65239
     */
```

Reply:
👍